draft-irtf-cfrg-hpke.md

```diff
@@ -379,17 +406,26 @@ following operations:
 
 - GenerateKeyPair(): Generate an ephemeral key pair `(skX, pkX)`
   for the DH group in use
+- DeriveKeyPair(ikm): Derive a key pair `(skX, pkX)` from
+  the byte string `ikm`, where `ikm` SHOULD have at least `Nsk` bytes
+  of entropy (see {{derive-key-pair}} for discussion)
 - DH(sk, pk): Perform a non-interactive DH exchange using the
   private key sk and public key pk to produce a Diffie-Hellman
   shared secret of length Ndh
 - Serialize(pk): Produce a byte string of length `Npk`
-  encoding the public key `pk`
+  encoding the public key `pk`.
```

Comment:
Some of these list items have a terminating period, and others don't. Can we add periods to all list items (here and above)?